workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/src/CodeGenerator.cs

```diff
@@ -104,6 +109,12 @@ private HashSet<string> ExtractEnums(ICollection<UnitySchemaFile> schemas)
             return enumSet;
         }
 
+        private List<string> ExtractWorkerTypes(string path)
+        {
+            var fileNames = Directory.EnumerateFiles(path, "spatialos.*.worker.json");
```

Comment:
What happens if the `path` is invalid?

Reply:
How will this play with FPL?

Reply:
Yeah this will need to be configurable

Reply:
Also the names don't have to follow that pattern - you will need to parse the JSON

Reply:
I am guessing the FPL handling can be done within the FPL epic, unless ☝️ would be trivial?

Reply:
Yes it would - but noting here as there might be a few places where we can make our lives easier in the future :)

Reply:
oh really, I thought they need to be called that? or is this part of FPL that it won't work for that anymore?

Reply:
interesting, will rewrite it